CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## 2014-10-31 Release 1.2.0
+## 2015-04-14 Release 1.3.0
+### Summary
+- Added support for Windows Systems
+
```

Comment:
CHANGELOG updates shouldn't be made in the pull request; this will be done at release time.